content/en/blog/README.md

```diff
@@ -0,0 +1,62 @@
+# Istio blog
+
+The Istio blog is owned by the [Steering committee](GET LINK) and run by the [Editorial Team](#leadership).
+
+This section covers documentation, processes, and roles for the [Istio blog](https://istio.io/latest/blog/2020/).
+
+## Leadership 
+
```

Comment:
can we indicate some of these are optional?

Reply:
this is stated in the section _Review process_